jbpm/pom.xml

```
@@ -20,13 +20,33 @@
   <properties>
   </properties>
 
-  <modules>
-    <module>jbpm-flow</module>
-    <module>jbpm-flow-builder</module>
-    <module>jbpm-bpmn2</module>
-    <module>jbpm-serverless-workflow</module>
-    <module>process-serialization-protobuf</module>
-    <module>process-workitems</module>
-  </modules>
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>jbpm-flow</module>
+        <module>jbpm-flow-builder</module>
+        <module>jbpm-bpmn2</module>
+        <module>jbpm-serverless-workflow</module>
+        <module>process-serialization-protobuf</module>
+        <module>process-workitems</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>prod</id>
+      <activation>
+        <property>
+          <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```